src/main/java/com/health/threadpool/TaskProcessingService.java

```diff
@@ -53,9 +65,319 @@ public class TaskProcessingService {
+
+        return st;
+    }
+
+    public Map<String, String> addDocument(String documentId, String documentType, String documentPath,
+            String documentUrl, int rank, String view_url, int languageId, String language,
+            Optional<Integer> categoryId, Optional<String> category, Optional<Integer> topicId, Optional<String> topic,
+            Optional<String> outlinePath, String requestType) {
+
+        Map<String, String> resultMap = new HashMap<>();
+
+        logger.info(
+                "RequestType:{} Language:{} View_URL: {} documentId:{} documentPath:{} documentType:{} outlinePath:{}",
+                requestType, language, view_url, documentId, documentPath, documentType, outlinePath);
+
+        if (documentPath == null) {
+
+            resultMap.put(CommonData.STATUS, CommonData.STATUS_FAILED);
+            resultMap.put(CommonData.REASON, "documentPath file does not exist");
+            return resultMap;
+
+        }
+
+        if (outlinePath != null && !outlinePath.isPresent()) {
+
+            resultMap.put(CommonData.STATUS, CommonData.STATUS_FAILED);
+            resultMap.put(CommonData.REASON, "outline file does not exist");
+            return resultMap;
+
+        }
+
+        QueueManagement queuemnt = new QueueManagement();
+
+        if (outlinePath != null && outlinePath.isPresent())
+            queuemnt.setOutlinePath(outlinePath.get());
+        queuemnt.setRequestTime(getCurrentTime());
+        queuemnt.setRequestType(requestType);
+        queuemnt.setDocumentId(documentId);
+        queuemnt.setDocumentType(documentType);
+        queuemnt.setDocumentPath(documentPath);
+        queuemnt.setDocumentUrl(documentUrl);
+        queuemnt.setRank(rank);
+        queuemnt.setViewUrl(view_url);
+        queuemnt.setLanguageId(languageId);
+        queuemnt.setStatus("pending");
+        if (language != null)
+            queuemnt.setLanguage(language);
+        if (category != null && category.isPresent())
+            queuemnt.setCategory(category.get());
+        if (categoryId != null && categoryId.isPresent())
+            queuemnt.setCategoryId(categoryId.get());
+        if (topic != null && topic.isPresent())
+            queuemnt.setTopic(topic.get());
+        if (topicId != null && topicId.isPresent())
+            queuemnt.setTopicId(topicId.get());
+
+        queueRepo.save(queuemnt);
+
+        resultMap.put(CommonData.QUEUE_ID, Long.toString(queuemnt.getQueueId()));
+        resultMap.put(CommonData.STATUS, CommonData.SUCCESS);
+
+        return resultMap;
+
+    }
+
+    public void addUpdateDeleteTutorial(Tutorial tutorial, String requestType) {
+
+        String documentId = CommonData.DOCUMENT_TYPE_TUTORIAL + tutorial.getTutorialId();
+        String documentType = CommonData.DOCUMENT_TYPE_TUTORIAL;
+        String documentPath = tutorial.getTimeScript();
+        String documentUrl = "";
+        if (tutorial.getConAssignedTutorial().getLan().getLanId() == 22)
```

Comment:
```suggestion
        ContributorAssignedTutorial conAssignedTutorial = tutorial.getConAssignedTutorial();
        if (conAssignedTutorial.getLan().getLanId() == 22)
```
Assign to a variable. Use the variable everywhere.